### PR TITLE
chore: update to ng10

### PR DESCRIPTION
**Pull request by aaron-steinfeld**

### CODE REVIEW

File: projects/assets-library/package.json [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-    "lodash": "^4.17.11"
+    "@angular/common": "^10.0.4",
+    "@angular/core": "^10.0.4",
+    "@angular/platform-browser": "^10.0.4",
+    "@angular/material": "^10.0.2",
+    "rxjs": "~6.6.0",
+    "lodash-es": "^4.17.15"
+  },
+  "dependencies": {
+    "tslib": "^2.0.0"
```
Comment (anandtiwary) [inline projects/assets-library/package.json:13]: Why is this not a peer dependency?

File: projects/distributed-tracing/src/shared/components/table/data-cell/metric/metric-table-cell-renderer.component.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 import {
-  TABLE_CELL_RENDERER_CELL_DATA,
-  TABLE_CELL_RENDERER_COLUMN_CONFIG,
-  TABLE_CELL_RENDERER_COLUMN_INDEX,
-  TABLE_CELL_RENDERER_ROW_DATA,
   TableCellAlignmentType,
   TableCellRenderer,
   TableCellRendererComponent,
-  TableColumnConfig
+  TableColumnConfig,
```
Comment (anandtiwary) [inline projects/distributed-tracing/src/shared/components/table/data-cell/metric/metric-table-cell-renderer.component.ts:7]: Why are these ordering changes showing up? Any changes in the default lint rule ?

File: projects/test-utils/package.json [hunk 3]
```diff
@@ -3,11 +3,14 @@
...
   "peerDependencies": {
-    "@angular/common": "^9.1.0",
-    "@angular/core": "^9.1.0",
-    "@angular/flex-layout": "^9.0.0-beta.29",
-    "@angular/platform-browser": "^9.1.0",
-    "@angular/router": "^9.1.0",
-    "rxjs": "~6.5.4"
+    "@angular/common": "^10.0.4",
+    "@angular/core": "^10.0.4",
+    "@angular/flex-layout": "^10.0.0-beta.32",
```
Comment (anandtiwary) [inline projects/test-utils/package.json:8]: I am surprised this one is still in beta

File: projects/distributed-tracing/src/shared/components/table/data-cell/metric/metric-table-cell-renderer.component.ts (hunk 2 above)
Comment (aaron-steinfeld) [reply projects/distributed-tracing/src/shared/components/table/data-cell/metric/metric-table-cell-renderer.component.ts:7]: prettier and tslint have disagreed on the alphabetical ordering wrt caps - I think that finally got resolved in https://github.com/palantir/tslint/pull/4064

File: projects/test-utils/package.json (hunk 3 above)
Comment (aaron-steinfeld) [reply projects/test-utils/package.json:8]: https://github.com/angular/flex-layout/issues/835

**Pull request merged**